server/utils/viewUtils.ts

```diff
@@ -13,6 +13,14 @@ export default class ViewUtils {
     return escape(val).val
   }
 
+  static nl2br(val: string): string {
+    // handle things like nunjucks macros (which are 'SafeStrings') that do not need formatting
+    if (typeof val !== 'string') return val
```

Comment:
Is it possible to write a test that expresses what this check is trying to achieve?

Reply:
sure - done